### ISSUE TITLE
VPin Studio Updater Modal Window does not close

### ISSUE THREAD

**Issue by adstotz** [syd711/vpin-studio#72]
VPin Studio prompted that a Version 2.6.0 was available. When clicking the button, there was a Server update available, but the client was already running v2.6.0. After the Server install was complete, the modal window remains on the screen but there is no way to close it. 

Workaround:
Right clicking on Taskbar to close program does not work. Had to stop the application with Task Manager where it is registered as "Zulu Platform x64 Architecture". 

A friendlier process name would be a good improvement as well as a close button on the update modal or have it auto-close when complete.

Screenshot:
![image](https://github.com/syd711/vpin-studio/assets/7671112/d8372542-c7b4-464b-914e-942ff014f4ba)

 - OS: Windows 11 Pro

**Comment by syd711** [comment]
Thanks for reporting, I'll look into it.

**Comment by syd711** [comment]
How did you manage to update the client before the server?
The fact that the dialog is not closing indicates that some error occured I haven catched properly. 
Could you attach the log files again? 

The workaround is to download the two .exe files from github directly. No other files are need when updating.
Sorry for the inconveniences!

**Comment by adstotz** [comment]
1. I downloaded the 2.6.0 exe installer andd ran it (there was no update button available in the VPin UI).
2. Opened VPin Studio (it now indicated that it was at v2.6.0)
3. Installed the service.
4. VPin then opened to a larger window and seemed connected to the service.
5. VPin Studio had the 2.6.0 upgrade button available so I clicked it.
6. The modal indicated that the client was already up to date, but that the server was not.

[vpin-studio-server.log](https://github.com/syd711/vpin-studio/files/13788641/vpin-studio-server.log)
[vpin-studio-ui.log](https://github.com/syd711/vpin-studio/files/13788643/vpin-studio-ui.log)
